sdk/eventhub/azure-eventhub/azure/eventhub/_configuration.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, **kwargs):
         self.custom_endpoint_hostname = None
         self.hostname = kwargs.pop("hostname")
         uamqp_transport = kwargs.pop("uamqp_transport")
+        amqp_library = PYAMQP_LIBRARY if not uamqp_transport else UAMQP_LIBRARY
+        self.user_agent = f"{amqp_library} {self.user_agent}"
```

Comment:
one other thing, not sure if this matters or not: do we want to make the space a dash here?
```suggestion
        self.user_agent = f"{amqp_library}-{self.user_agent}"
```

b/c I see in the previous user agent, there are no spaces so we may want to follow the same convention

Reply:
yeah this is one I was unsure about since the final string looks like this `f"{USER_AGENT_PREFIX}/{VERSION} {framework} ({platform_str})"`

so maybe it should be `f"{self.user_agent}/{amqp_library}"` making the final string `f"{self.user_agent}/{amqp_library}/{VERSION} {framework} ({platform_str})"`?

Reply:
I assume we're following the [HTTP user agent formatting convention](https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy)?
If so, it would be a list of packages paired with their versions, ordered from outer-most to inner-most - so something like:
`{eventhub-package}/{eventhub-version} {amqp-package}/{amqp-version} ...`

Reply:
I made some changes to bring about the transport library name & version from the transport classes itself, basically restoring  the Config class. 

The current user agent string is `'azsdk-python-eventhubs/5.11.0 Python/3.9.9 (Windows-10-10.0.22621-SP0)'`

After this change it will be `'azsdk-python-eventhubs/5.11.0 pyamqp/2.0.0a1 Python/3.9.9 (Windows-10-10.0.22621-SP0)'` or `'azsdk-python-eventhubs/5.11.0 uamqp/1.6.3 Python/3.9.9 (Windows-10-10.0.22621-SP0)'`